pkg/segment/results/segresults/segresults.go

```diff
@@ -374,17 +374,27 @@ func (sr *SearchResults) UpdateSegmentStats(sstMap map[string]*structs.SegStats,
 			}
 			continue
 		case utils.List:
+			if measureAgg.ValueColRequest != nil {
+				err := aggregations.ComputeAggEvalForList(measureAgg, sstMap, sr.segStatsResults.measureResults, sr.runningEvalStats)
+				if err != nil {
+					return fmt.Errorf("UpdateSegmentStats: qid=%v, err: %v", sr.qid, err)
+				}
+				continue
+			}
+
 			// Splunk documentation specifies that if more than 100 values are in the field, only the first 100 are returned.
 			strList := make([]string, 0, 100)
 
 			if currSst != nil && currSst.StringStats != nil && currSst.StringStats.StrList != nil {
+				sort.Strings(currSst.StringStats.StrList)
 				strList = utils.AppendWithLimit(strList, currSst.StringStats.StrList, 100)
 			}
 
 			if sr.runningSegStat[idx] != nil && sr.runningSegStat[idx].StringStats != nil {
+				sort.Strings(sr.runningSegStat[idx].StringStats.StrList)
 				strList = utils.AppendWithLimit(strList, sr.runningSegStat[idx].StringStats.StrList, 100)
 			}
-
+			sort.Strings(strList)
```

Comment:
Why do you sort here? I had the same question on the original PR that added `list`: https://github.com/siglens/siglens/pull/1379#discussion_r1714424294

Am I missing something? I think you're sorting somewhere else as well

Reply:
There is no need to sort.
I got confused by:
> Add the stats command with the list function to the search. The numbers are returned in ascending order in a single, multivalue result. 

Removed sort in latest commit.